NBitcoin.Tests/AltcoinTests.cs

```diff
@@ -241,6 +251,7 @@ public void CorrectCoinMaturity()
 			using (var builder = NodeBuilderEx.Create())
 			{
 				var node = builder.CreateNode();
+				this.EnsureWallet(node);
```

Comment:
Don't do this, you are putting burden on everybody making a test, and not only on this project. 

`NodeDownloadData.AdditionalRegtestConfig` can be used if you need to pass more config stuff.

Reply:
Unfortunately configuration to do so does not yet exist in Stratis. I created an issue in Stratis github as linked below.